Formula/elvish.rb

```diff
@@ -1,8 +1,10 @@
 class Elvish < Formula
   desc "Novel UNIX shell written in Go"
   homepage "https://github.com/elves/elvish"
-  url "https://github.com/elves/elvish/archive/0.10.tar.gz"
-  sha256 "1b053abd3f8c3b436712597e5dc7ce44c71737bdc0016cbf773a4e74c712851a"
+  url "https://github.com/elves/elvish.git",
+      :tag => "0.10.1",
```

Comment:
Why this change? If building from the archive still works, we should not switch.

Reply:
I thought this style is more preferred.. ok to revert

Reply:
In order of preference:
1. upstream release tarball
2. source archive tarball
3. tag + revision

Reply:
fixed